### DISCUSSION TITLE
Is there an api documentation?

### DISCUSSION THREAD

**Discussion by andykaufseo** [fishaudio/fish-speech#720] [General]
I have installed fishspeech 1.5, inference works ok in the webui (haven't tested via cli) - and also i'm able to start the api on port 8080 - but there's no documentation for how to use the api (or at least i couldn't find it anywhere).

any help would be appreciated

**Comment by None1145** [comment]
> I have installed fishspeech 1.5, inference works ok in the webui (haven't tested via cli) - and also i'm able to start the api on port 8080 - but there's no documentation for how to use the api (or at least i couldn't find it anywhere).
> 
> any help would be appreciated

https://github.com/fishaudio/fish-speech/blob/main/docs%2Fen%2Finference.md
